designsafe/static/scripts/data-depot/components/data-depot-listing/files-listing.template.html

```diff
@@ -80,10 +80,11 @@
                     <td data-ng-if="!item.lastModified"class="test-date-missing">--</td>
                     <td data-ng-if="$ctrl.operationLabel" class="test-op-label">
                         <button class="btn btn-primary test-op-btn" ng-if="item.type==='dir'" ng-click="$ctrl.operation({file: item})">{{$ctrl.operationLabel}}
-
-                        </button></td>
+                        </button>
+                    </td>
                     <td data-ng-if="!$ctrl.FileListingService.listings.main.loading && $ctrl.hasNotebooks()">
```

Comment:
The ticket for this issue says the link is broken, are we removing the link entirely or are we supposed to restore the functionality? If we're removing this, we may want to remove the table column as well.

Reply:
Good point. I'm not sure if that column is used for anything else. I'll poke around and check. If it's not used, I'll remove it.